docs/core/diagnostics/dumps.md

```diff
@@ -1,79 +1,47 @@
 ---
 title: Dumps - .NET
 description: An introduction to dumps in .NET.
-ms.date: 10/31/2022
+ms.date: 01/11/2023
 ---
 
 # Dumps
 
-A dump is a file that contains a snapshot of the process at the time it was created and can be useful for examining the state of your application. Dumps can be used to debug your .NET application when it is difficult to attach a debugger to it such as production or CI environments. Using dumps allows you to capture the state of the problematic process and examine it without having to stop the application.
+A dump is a file that contains a snapshot of the process at the time the dump was created and can be useful for examining the state of your application. Dumps can be used to debug your .NET application when it is difficult to attach a debugger to it, such as production or CI environments. Using dumps allows you to capture the state of the problematic process and examine it without having to stop the application.
 
 ## Collect dumps
 
-Dumps can be collected in a variety of ways depending on which platform you are running your app on.
+Dumps can be collected in a variety of ways depending on which platform your app is running on.
 
 > [!NOTE]
 > Dumps may contain sensitive information because they can contain the full memory of the running process. Handle them with any security restrictions and guidances in mind.
 
 > [!TIP]
 > For frequently asked questions about dump collection, analysis, and other caveats, see [Dumps: FAQ](faq-dumps.yml).
 
-### Collect dumps on crash
+* You can use environment variables to configure your application to [collect a dump on a crash](collect-dumps-crash.md).
 
-You can use environment variables to configure your application to collect a dump upon a crash. This is helpful when you want to get an understanding of why a crash happened. For example, capturing a dump when an exception is thrown helps you identify an issue by examining the state of the app when it crashed.
+* You may want to collect a dump when the app hasn't crashed yet. For example, if you want to examine the state of an application that seems to be in a deadlock, configuring the environment variables to collect dumps on crash will not be helpful because the app is still running.
 
-The following table shows the environment variables you can configure for collecting dumps on a crash.
+* [dotnet-dump](dotnet-dump.md) is a simple cross-platform command line tool to collect a dump. [dotnet-monitor] is an alternative if you need to collect a dump remotely or with automated triggering conditions. Several other debugger tools such as [Visual Studio](/visualstudio-docs/docs/debugger/using-dump-files) or [windbg](/windows-hardware/drivers/debugger/-dump--create-dump-file-) also have dump collection features.
```

Comment:
Nit: maybe we don't need to mention dotnet-monitor here if we are going to mention it again in the next bullet?